src/System.ServiceModel.Primitives/tests/ServiceModel/DuplexClientBaseTest.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.Threading.Tasks;
+using Xunit;
+
+public class DuplexClientBaseTest
+{
+    [Fact]
+    public static void DuplexClientBase_Ctor_Initializes_State()
+    {
+        InstanceContext context = new InstanceContext(new WcfDuplexServiceCallback());
+        Binding binding = new NetTcpBinding();
+        EndpointAddress endpoint = new EndpointAddress(Endpoints.Tcp_NoSecurity_Callback_Address);
+        MyDuplexClientBase<IWcfDuplexService> duplexClientBase = new MyDuplexClientBase<IWcfDuplexService>(context, binding, endpoint);
+
+        Assert.Equal<EndpointAddress>(endpoint, duplexClientBase.Endpoint.Address);
+        Assert.Equal<CommunicationState>(CommunicationState.Created, duplexClientBase.State);
+
+        duplexClientBase.Abort();
+    }
```

Comment:
The prevailing wisdom has been to Close() every channel, thus freeing up resources ASAP. In the case of testing, it probably doesn't matter that we don't close and abort everything since the server is short-lived and the client probably isn't long-lived. However, I'm working on the assumption that *every piece of code that we put into test is a potential code sample* . 

Thus, if we think it's good practice for production, we should, to the best of our ability, include this in our own best practices, even if it's "just" test code.